py3status/modules/imap.py

```diff
@@ -42,13 +43,19 @@ def check_mail(self, i3s_output_list, i3s_config):
 
     def _get_mail_count(self):
         try:
+            mail_count = 0
+            directories = self.mailbox.split(',')
             connection = imaplib.IMAP4_SSL(self.imap_server, self.port)
             connection.login(self.user, self.password)
-            connection.select(self.mailbox)
-            unseen_response = connection.search(None, self.criterion)
-            mails = unseen_response[1][0].split()
-            mail_count = len(mails)
+            
+            for directory in directories:
+                connection.select(directory)
+                unseen_response = connection.search(None, self.criterion)
+                connection.close()
```

Comment:
little glitch here and we're done, should be closed outside the `for`, just before the return I guess plz